pkg/skyenv/values_darwin.go

```diff
@@ -6,13 +6,14 @@ package skyenv
 const (
 	//OS detection at runtime
 	OS = "mac"
-
 	// SkywirePath is the path to the installation folder.
 	SkywirePath = "/Library/Application Support/Skywire"
+	// Ptext is the description to use for the -p flag for config gen and for the visor
+	Ptext = "use mac installation path: " + SkywirePath
```

Comment:
Seems redundant to me. Can we just use the SkwirePath var we already export and the OS we can get from the stdlib?